### PR TITLE
Introduce Character.utf8

### LINKED ISSUE #2604: Add `Character.utf8` similar to `String.utf8` field
### Issue to be solved

`Character` lacks a way to get the byte array of the UTF-8 encoding.

### Suggested Solution

String has the field `utf8` to get the byte array of the UTF-8 encoding.

Add a similar field to `Character`. Suggested in https://github.com/onflow/cadence/issues/1972

### PR DESCRIPTION
**Pull request by darkdrag00nv2**
Closes #2604

## Description

Introduce `Character.utf8` similar to `String.utf8`.

Docs PR will follow soon.

______

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: runtime/interpreter/value.go [hunk 1]
```diff
@@ -984,6 +984,9 @@ func (v CharacterValue) GetMember(interpreter *Interpreter, _ LocationRange, nam
 				)
 			},
 		)
+
+	case sema.CharacterTypeUtf8FieldName:
+		return ByteSliceToByteArrayValue(interpreter, []byte(v))
```
Comment (turbolent) [inline runtime/interpreter/value.go:989]: The allocation of the byte slice must be metered above

**Review by turbolent — COMMENTED** [review]
Nice!

File: runtime/interpreter/value.go (hunk 1 above)
Comment (darkdrag00nv2) [reply runtime/interpreter/value.go]: Done, added.

File: runtime/interpreter/value.go [hunk 2]
```diff
@@ -984,6 +984,11 @@ func (v CharacterValue) GetMember(interpreter *Interpreter, _ LocationRange, nam
 				)
 			},
 		)
+
+	case sema.CharacterTypeUtf8FieldName:
+		b := []byte(v)
+		common.UseMemory(interpreter, common.NewBytesMemoryUsage(len(b)))
+		return ByteSliceToByteArrayValue(interpreter, b)
```
Comment (turbolent) [inline runtime/interpreter/value.go:991]: The allocation has to be metered before:

```suggestion
		common.UseMemory(interpreter, common.NewBytesMemoryUsage(len(v)))
		return ByteSliceToByteArrayValue(interpreter, []byte(v))
```

**Review by turbolent — APPROVED** [review]
Nice!

**Review by dsainati1 — APPROVED** [review]

**Pull request merged**